primitives/derive/src/numeric.rs

```diff
@@ -6,8 +6,17 @@ use quote::quote;
 
 pub fn numeric_impl(input: TokenStream) -> Result<TokenStream> {
     let input = input.to_string();
-    let numeric = ::iroha_numeric::Numeric::from_str(&input)
+
+    let mut input = input.as_str();
+    for pattern in ["_u8", "_u16", "_u32", "_u64", "_u128"] {
+        if let Some(stripped) = input.strip_suffix(pattern) {
+            input = stripped;
+        }
+    }
```

Comment:
Why do you want to allow suffixes in macro input?
Imo it looks redundant and doesn't cover all cases (rust allows to write `10_u32` and `10u32`).

Reply:
should I remove it?